locale/fi/LC_MESSAGES/django.po

```diff
@@ -3610,3 +3671,1254 @@ msgstr "Unohda automaattinen uudelleenohjaus."
 #: userprofile/views.py:80
 msgid "No privacy notice. Please notify administration!"
 msgstr "Ei tietosuojailmoitusta. Ota yhteys ylläpitäjään!"
+
+#: venv/lib/python3.5/site-packages/colorfield/fields.py:10
```

Comment:
You have run `manage.py makemessages` in a directory that contained modules that do not belong to A+. This has messed up this po file. There should not be any string from any `venv` directory here. They are not part of A+. They must be removed.

Reply:
I removed the venv strings myself in the new branch: https://github.com/Aalto-LeTech/a-plus/commits/feature-exam-app

Reply:
Done. (by Mankro)